### DISCUSSION TITLE
Put game server version number on server placard.

### DISCUSSION THREAD

**Discussion by Loops117** [gameserverapp/Platform#852] [Suggestions]
I have difficulty finding which game version my game server is running. In fact, i don't think there's a way to find that detail without connecting to the server in-game. I am also left to force update a server just to make sure it's running the most recent version.

Can we add the current game server version each server is running on the placard and within the dashboard for each server?

**Comment by birdman3131** [comment]
So it is shown for ASE but not ASA.

For ASA you can look under logs > Shootergame.log > in the first 10 lines should be something along the lines of 
[2024.05.22-10.01.14:418][ 0]ARK Version: 43.12

**Reply by Loops117** [reply]
I appreciate the help! I will look at the logs for now, but hopefully we can get the version on the front like it does for ASE.

Thank you
